jreleaser cannot deploy to Central

When releasing 3.16.0 with the new workflow, that uses jreleaser, the following happens:

Citing from https://github.com/quarkiverse/quarkus-cxf/actions/runs/11498024409/job/32002959293
```
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.2661438Z ☕ java -jar jreleaser-cli.jar deploy
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.5789965Z jreleaser 1.15.0-SNAPSHOT. Consider becoming a sponsor at https://opencollective.com/jreleaser
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.6034918Z [INFO]  JReleaser 1.15.0-SNAPSHOT
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.6041437Z [INFO]  Configuring with jreleaser.yml
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.6044305Z [INFO]    - basedir set to /home/runner/work/quarkiverse-release/quarkiverse-release
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.6047724Z [INFO]    - outputdir set to /home/runner/work/quarkiverse-release/quarkiverse-release/out/jreleaser
deploy_to_central	Deploy to Central	2024-10-24T11:11:09.6129010Z [INFO]  Reading configuration
deploy_to_central	Deploy to Central	2024-10-24T11:11:10.2138664Z [INFO]  git-root-search set to false
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.3308639Z [INFO]  Loading variables from /home/runner/.jreleaser/config.properties
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.3312109Z [WARN]  Variables source /home/runner/.jreleaser/config.properties does not exist
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.3327165Z [INFO]  Validating configuration
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.3329293Z [INFO]  Strict mode set to false
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4079064Z [INFO]  Project version set to 3.16.0
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4081388Z [INFO]  Release is not snapshot
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4083815Z [INFO]  Timestamp is 2024-10-24T11:11:10.164813252Z
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4086078Z [INFO]  HEAD is at f1f3a7d
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4088290Z [INFO]  Platform is linux-x86_64
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4108460Z [INFO]  dry-run set to false
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4125469Z [INFO]  Deploying Maven artifacts
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4127203Z [INFO]    [maven] Deploying all staged artifacts
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4197165Z [INFO]      [nexus2] Deploying to maven-central
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.4199245Z [INFO]      [nexus2] nexus2 set to FULL_DEPLOYMENT
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.6971018Z [INFO]      [nexus2] Verifying prerequisites
deploy_to_central	Deploy to Central	2024-10-24T11:11:13.7005527Z [INFO]      [nexus2] Verifying POMs
Error: to_central	Deploy to Central	2024-10-24T11:13:04.6675130Z [ERROR]     [nexus2] quarkus-cxf-xjc-plugins-deployment-3.16.0-sources.jar is missing
deploy_to_central	Deploy to Central	2024-10-24T11:13:04.6680609Z [INFO]  Writing output properties to out/jreleaser/output.properties
Error: to_central	Deploy to Central	2024-10-24T11:13:04.6765332Z [ERROR] JReleaser failed after 01:51 m
deploy_to_central	Deploy to Central	2024-10-24T11:13:04.6778534Z Rules for publishing to Maven Central were not met
Error: Process completed with exit code 1.
```

See also 

* https://github.com/kordamp/pomchecker/issues/10
* https://github.com/kordamp/pomchecker/issues/11

--- a/extensions/xjc-plugins/deployment/src/main/resources/workaround-issue-1567.txt
+++ b/extensions/xjc-plugins/deployment/src/main/resources/workaround-issue-1567.txt
@@ -1,2 +1,0 @@
-This file should cause that -source.jar and -javadoc.jar are created.
-See https://github.com/quarkiverse/quarkus-cxf/issues/1567

